### PR TITLE
Clarify that releases on weekend are up to release captain 🚢 

### PR DESCRIPTION
**Pull request by daxmc99**
Give more power to the release captain to move the monthly release date to account for holidays / non-working days.

### CODE REVIEW

**Comment by pecigonzalo** [comment]
Sounds great! I would share in #dev-chat and @ someone from CE and Product to review here.

**Comment by emchap** [comment]
I don't see any issue with this, though I think long-term we might want to standardize on a date (like, always the first working day back on the 20th or something). From a process level seems like a totally reasonable work/life balance change.

**Comment by Joelkw** [comment]
I am not super familiar with our site admin personas or relationship on how distribution works, but from product I'd frame this question around the user. Some quick example scenarios:

- if the people in charge of updating their instance wait for a release to drop explicitly / a slack ping, it probably doesn't matter when we release since we manually communicate it. 
- if those stakeholders like to schedule something regularly into their calendar to remind themselves when to update, it might even be more helpful for them if we explicitly set our release cadence to the 7-day week rather than a month-date? (Like, every 4th Tuesday or whatever; this way it's never on a weekend anyway) 
- if those stakeholders just generally update "after the 20th" then it might make most sense to *only* shift this day forward rather than forward *or* back, so the 20th is always the latest possible day and is easy to remember? 

Are any of those situations more/less applicable to this? 

I'm not worried about this change impacting internal product/eng teams' planning; no one depends on having exactly 1-month between releases.

File: handbook/engineering/releases/index.md
Comment (Joelkw) [inline handbook/engineering/releases/index.md:134]: Maybe add some minimum time this announcement must be made just for clarity? Like 2 weeks ahead?

**Comment by emchap** [comment]
Of the paying customers that are on 3.25.0 now, we have: 

- Tradera at Paypal on Team
- Tinder (we manage them)
- Prezi
- Fiverr
- Qualtrics

So it feels like a relatively small customer impact. I like the idea of keeping it consistent on a day (fourth Thursday or whatever) because it allows us to always avoid Friday releases if we want to and can be scheduled into calendar software more easily.

**Review by virginiaulrich — APPROVED** [review]
I think this is a very good and healthy change!

File: handbook/engineering/releases/index.md
Comment (christinaforney) [inline handbook/engineering/releases/index.md:133]: Why +/- 3 and not +/- 2? and/or why not the previous Friday or following Monday? Just curious to hear understand the thought process  :) (I do not feel strongly about this and support as written).

**Review by christinaforney — APPROVED** [review]
Agree with @emchap about standardizing this shift for more consistency, but don't feel like we need to do that at this time. +1 to work/life balance!

**Review by pecigonzalo — APPROVED** [review]

**Comment by pecigonzalo** [comment]
How about moving to `-` days, and guarantee that the release is going to be there on or by the 20th? Similar to what @emchap suggested.

File: handbook/engineering/releases/index.md
Comment (pecigonzalo) [reply handbook/engineering/releases/index.md:134]: We no longer announce releases in advance as part of the [single-day release](https://github.com/orgs/sourcegraph/projects/90) project.

If we guarantee that by the 20th the release is there, would there be a need for an announcement? Basically, if 20th falls on the weekend/holiday, we cut the last working day before that, so by the 20th, it's there and the blog can be sent the next working day after the 20th.

File: handbook/engineering/releases/index.md
Comment (daxmc99) [reply handbook/engineering/releases/index.md:133]: My thought here was if the release was on a Monday holiday after the weekend we could release on the Thursday / Friday before.

**Comment by daxmc99** [comment]
Kept the wording that we release by the 20th and allowed the release captain to move the day up to 3 days **before** the 20th to ensure we always release by then. 
Planning to merge but I'm happy to reopen if new concerns crop up.

**Review by slimsag — APPROVED** [review]
LGTM

**Pull request merged**
